### PR TITLE
Remove erroneous LOG message.

### PR DESCRIPTION
**Pull request by psudheer21**
Removed the erroneous LOG message.
Changed the running time from 30 seconds to 300 seconds by default.

### CODE REVIEW

File: config/workload_all.xml [hunk 1]
```diff
@@ -33,7 +33,7 @@
     <useThinkTime>true</useThinkTime>
     <works>
         <work>
-          <time>30</time>
+          <time>300</time>
```
Comment (rkarthik007) [inline config/workload_all.xml:36]: Hey @psudheer21 - this changes the think time from 30 to 300, though the title is only about log message removal. Is this intentional? If so, could we change the title and also add a comment on top referencing the section of the spec, etc just as with the number of terminals above?

Comment (psudheer21) [reply config/workload_all.xml:36]: It is intentional.
There is no time factor in the spec. 
ITs just that we need to run it sufficiently long to get proper results..

Comment (rkarthik007) [reply config/workload_all.xml:36]: Got it. Would be great to add a comment on top saying we're running the workload for 300 seconds by default.

**Review by rkarthik007 — APPROVED** [review]
LGTM!

**Pull request merged**